README.md

```diff
@@ -48,7 +48,7 @@ AtomOne is a relatively simple and vanilla Cosmos SDK chain with minor modificat
 
 The following public RPC and API endpoints are available:
 
-- https://atomone-api.allinbits.com/
+- https://atomone-api.allinbits.com/ <!-- markdown-link-check-disable-line -->
```

Comment:
Why is the API URL excepted but not the following RPC URL?

Reply:
Check the links you will see that the api returns a 501 at root.